### ISSUE TITLE
Profiles for EV need to be generated for 20 kWh storage volumes

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#808]
The EV volume has been updated in the ETM from 25 to 20 kWh.

**Comment by ChaelKruip** [comment]
Assigned @jorisberkhout

**Comment by jorisberkhout** [comment]
There is more to this issue than I thought. The `20 kWh` in the original post comes from research that @mariatsagkaraki did (see [the relevant node source analysis](https://github.com/quintel/etdataset/blob/master/nodes_source_analyses/transport/transport_car_using_electricity.converter.xlsx)). This `20 kWh` is the average storage volume for a number of small EV, all having storage volumes close to this `20 kWh`. As @AlexanderWirtz rightly asked in a discussion we had about this, what about the Tesla Model S? The Model S has a storage volume of `60 kWh` or `85 kWh` depending on the specific type, in both cases much higher than the `20 kWh`.

Currently, the NL EV market is dominated by the Tesla Model S, see the table below which is based on data from [RVO](http://www.rvo.nl/onderwerpen/duurzaam-ondernemen/energie-en-milieu-innovaties/elektrisch-rijden/stand-van-zaken/cijfers) and the car manufacturers specifications (I only included cars for person transport, not the small vans and the quadricycle that are in the original data).

| Model | Storage Volume | Current number in NL (31 January 2016) |
|:--- | ---:| ---:|
| Tesla Model S | 60 or 85 | 4469 |
| Nissan Leaf | 30 | 1345 |
| Renault Zoe | 22 | 1135	 |
| BMW I3 | 18.7 | 543 |
| Smart ForTwo / Electric Drive |16.5 | 480 |
| Nissan E-VN200 | 24 | 214 |
| Volkswagen UP | 18.7 | 158 |
| **Total** | | **8344** |

The weighted average for the storage volume range between `43 kWh` and `56 kWh` (for a storage volume for the Tesla Model S of `60 kWh` and `85 kWh` respectively). This seems to suggest that we should increase the storage volume for EV in the ETM. However, given the large spread in storage volumes, I would rather discuss with an expert first, before taking the weighted average, which will probably evolve strongly in the future.

Long story short, I would suggest to do two things:
1. Change the capacity of the EV in the ETM back to `25 kWh`. This is very little work and will avoid having to generate new profiles.
2. Talk to experts on which typical storage volume to use for EV in the ETM and update the profiles accordingly.

If you agree, @ChaelKruip and @AlexanderWirtz , I will implement number 1 and talk to you about number 2.

**Comment by ChaelKruip** [comment]
> If you agree, @ChaelKruip and @AlexanderWirtz , I will implement number 1 and talk to you about number 2.

:+1:

**Comment by AlexanderWirtz** [comment]
:+1:
